test/test_client.py

```diff
@@ -393,10 +393,10 @@ def test_uri_option_precedence(self):
     def test_connection_timeout_ms_propagates_to_DNS_resolver(self):
         # Patch the resolver.
         from pymongo.srv_resolver import resolver
-        patched_resolver = FunctionCallRecorder(resolver.query)
-        pymongo.srv_resolver.resolver.query = patched_resolver
+        patched_resolver = FunctionCallRecorder(resolver.resolve)
```

Comment:
This needs to be updated now that we use `pymongo.srv_resolver._resolve` instead of `pymongo.srv_resolver.resolver.resolve`.

Reply:
Let me know if I did this correctly.